app/knight.py

```diff
@@ -0,0 +1,25 @@
+"""
+Here we introduce class Knight and enable its instances to fight each other.
+"""
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self,

```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one,